### DISCUSSION TITLE
Tutorial Error no module name parcels

### DISCUSSION THREAD

**Discussion by cuchisavila** [Parcels-code/Parcels#1187] [General]
Hi!
Just getting started and I got this error in the tutorial
-
--------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Input In [5], in <cell line: 1>()
----> 1 from parcels import FieldSet, ParticleSet, JITParticle, AdvectionRK4
      3 # 1. Setting up the velocity fields in a FieldSet object
      5 fname = 'GlobCurrent_example_data/*.nc'

ModuleNotFoundError: No module named 'parcels'

What should I do?
I am running it in jupyter notebook
![Screen Shot 2022-06-22 at 1 06 20 PM](https://user-images.githubusercontent.com/39143475/175126587-b7a5eae8-a747-456b-a4df-10662aa3945e.png)

**Comment by erikvansebille** [comment]
Hi @cuchisavila. This Error suggests that Parcels hasn't correctly been installed, so that it was not found by the python interpreter... Could you try installing Parcels again by carefully following all the 7 steps at https://oceanparcels.org/#installing?

**Reply by cuchisavila** [reply]
I installed it again,  but I am still getting the same error. Also how do I know if it was installed correctly?
I have followed step 1-5 and so the test 

```
_```
(py3_parcels) Bobbys-iMac:parcels_examples ROMs$ python example_peninsula.py --fieldset 100 100
INFO: Generating FieldSet output with basename: peninsula
WARNING: Particle initialisation from field can be very slow as it is computed in scipy mode.
Peninsula: Advecting 20 particles for 1 day, 0:00:00
INFO: Compiled ArrayMyParticleAdvectionRK4UpdateP ==> /var/folders/wz/ntgxy4s15mb5j51s8rn_3rch0000gn/T/parcels-501/libc9f9b4bc0e53614be4742f5f245f1094_0.so
(py3_parcels) Bobbys-iMac:parcels_examples ROMs$
```_ 
```


steps 6-7 seem to be to open jupyter notebooks but not part of the installation. 
how do I know if the example ran correctly?
how do I know if it is installed correctly?
I got no errors during installation

**Reply by erikvansebille** [reply]
OK, it seems that the installation went well. So another possibility why your Jupyter doesn't recognise your parcels installation, is because it uses another Python environment. I don't know how you installed Jupyter, but you need to make sure to point it to the correct conda-environment, otherwise the python interpreter won't know where the parcels module files are located. 
Perhaps you can ask an IT administrator at your institute for help in setting up you python and Jupyter environment?
